packages/storage/src/task.ts

```diff
@@ -344,7 +369,8 @@ export class UploadTask {
         requestInfo,
         newTextConnection,
         authToken,
-        appCheckToken
+        appCheckToken,
+        false
```

Comment:
why false?

Reply:
I added a comment above, but basically we don't want to retry any of the uploads with backoff immediately since we want to make a query request first before retrying

Reply:
Actually, nevermind. It should be true. Nice catch